app/views/profiles/personal_access_tokens/index.html.erb

```diff
@@ -65,6 +65,7 @@
           count: @active_access_tokens.count
         ) %>
       </h2>
+      <div id="personal-access-tokens-revoke-errors"></div>
```

Comment:
Why did you add an empty div?

Reply:
The empty div is for turbostream to display any errors returned when revoking a personal access token

Reply:
If that is the case, then define it as follows:
`<%= turbo_frame_tag "personal-access-token-revoke-errors" %>`

Reply:
Updated in 5b0317446b5f8e0cd4ae4a600a9a28e48d3a74ce